src/main/java/bio/terra/service/filedata/azure/util/BlobContainerCopier.java

```diff
@@ -22,6 +38,9 @@ public class BlobContainerCopier {
   private static final Logger logger = LoggerFactory.getLogger(BlobContainerCopier.class);
   private static final Duration DEFAULT_SAS_TOKEN_EXPIRATION = Duration.ofHours(24);
   private final BlobContainerClientFactory destinationClientFactory;
+  private static final String USER_PROJECT_QUERY_PARAM = "userProject";
+  // Signing URL for 48 hours to handle large file transfers
+  private static final long SIGNED_URL_DURATION_MINUTES = TimeUnit.HOURS.toMinutes(48);
```

Comment:
Would this be something that we'd prefer to have config control over?

Reply:
Yeah, that's a great call